### ISSUE TITLE
alignak-backend-cli do not allow to create a service for a different host

### ISSUE THREAD

**Issue by mohierf** [Alignak-monitoring-contrib/alignak-backend-client#50]
When a service with the same name still exists, the `alignak-backend-cli` script do not allow to create a service with the same name but for a different host.

**Comment by ddurieux** [comment]
The description of this issue is not understandable :(

**Comment by mohierf** [comment]
Yes it is. When a service with the same name exists ... allow to create a new service if it is a service for a different host than the existing one :)

**Comment by mohierf** [comment]
I merged rapidly because it is a really annoying bug to import information into the backend :/

**Comment by ddurieux** [comment]
It could be cool I review it and add tests :(

**Comment by mohierf** [comment]
Sure. But I neede rapidly a pypi version ... tests are on my server and are
coming soon ;)

2017-10-01 20:47 GMT+02:00 David Durieux <notifications@github.com>:
